### ISSUE TITLE
[package] gst-plugins-base/1.19.2: Not compatible with conan 2

### ISSUE THREAD

**Issue by ryankav** [conan-io/conan-center-index#25158]
### Description

The recipe doesn't seem to be compatible with conan 2

### Package and Environment Details

* Package Name/Version: **gst-plugins-base/1.19.2**
* Operating System+version: **Linux Ubuntu 20.04**
* Compiler+version: **clang 14**
* Docker image: **nvidia/cuda:12.6.0-cudnn-runtime-ubuntu20.04**
* Conan version: **conan 2.7.0**
* Python version: **Python 3.10.14**

### Conan profile

[conf]
tools.system.package_manager:mode = install

[settings]
arch=x86_64
build_type=Release
compiler=clang
compiler.cppstd=gnu14
compiler.libcxx=libstdc++11
compiler.version=14
os=Linux

[buildenv]
LDFLAGS=-fuse-ld=lld

[runenv]
LDFLAGS=-fuse-ld=lld

### Steps to reproduce

add `gst-plugins-base/1.19.2` to the conanfile.txt and try and build.

### Logs

<details><summary>Click to expand log</summary>

```
*********************************************************
Recipe 'gst-plugins-base/1.19.2' seems broken.
It is possible that this recipe is not Conan 2.0 ready
If the recipe comes from ConanCenter, report it at https://github.com/conan-io/conan-center-index/issues
If it is your recipe, check if it is updated to 2.0
*********************************************************

ERROR: Package 'gst-plugins-base/1.19.2' not resolved: gst-plugins-base/1.19.2: Cannot load recipe.
Error loading conanfile at '/root/.conan2/p/gst-pe2054fd2c8e73/e/conanfile.py': Unable to load conanfile in /root/.conan2/p/gst-pe2054fd2c8e73/e/conanfile.py
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/root/.conan2/p/gst-pe2054fd2c8e73/e/conanfile.py", line 1, in <module>
    from conans import ConanFile, tools, Meson, VisualStudioBuildEnvironment
ImportError: cannot import name 'ConanFile' from 'conans' (/opt/conan/lib/python3.10/site-packages/conans/__init__.py)
.
```

</details>

**Comment by ryankav** [comment]
Am happy to issue a PR to fix this and the other plugin recipes if wanted. Assume I would follow the example set in this PR https://github.com/conan-io/conan-center-index/pull/13400

**Comment by valgur** [comment]
See
- #19530
- #20992

The GStreamer plugin recipes are blocked by several missing binaries, which will probably mostly be resolved once the PRs related to #25090 get merged.

**Comment by ryankav** [comment]
Will close the issue as clearly a duplicate
